services/dbaas/dbaas_client.go

```diff
@@ -68,6 +68,11 @@ func (c *Client) DeleteXtraDBCluster(ctx context.Context, in *controllerv1beta1.
 	return c.xtradbClusterClient.DeleteXtraDBCluster(ctx, in, opts...)
 }
 
+// RestartXtraDBCluster restarts XtraDB cluster.
+func (c *Client) RestartXtraDBCluster(ctx context.Context, in *controllerv1beta1.RestartXtraDBClusterRequest, opts ...grpc.CallOption) (*controllerv1beta1.RestartXtraDBClusterResponse, error) {
```

Comment:
@askomorokhov @BupycHuk It is probably too late for this PR, but why we pass `opts ...grpc.CallOption`?

Reply:
Just to keep ability to pass them. it's not required at the moment.

Reply:
Then I don't quite understand why we need that wrapper that just passes all parameters as-is, without any logic, adoption, etc.

Let's drop opts once we have a chance, and then maybe revisit if we need that wrapper at all

Reply:
OK, I'll try to remove opts here.

Reply:
Only if it can be done quickly in that PR

Reply:
It requires many changes in many places. Better to do this without hurrying.